shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/show/executor/ExportSchemaConfigurationExecutor.java

```diff
@@ -0,0 +1,348 @@
+        result.append(indent(4)).append("rateLimiter").append(COLON_NEWLINE);
+        result.append(indent(5)).append("type").append(COLON_SPACE).append(rateLimiterConfig.getType()).append(NEWLINE);
+        if (null != rateLimiterConfig.getProps() && !rateLimiterConfig.getProps().isEmpty()) {
+            result.append(indent(5)).append("props").append(COLON_NEWLINE);
+            for (Map.Entry<Object, Object> each : rateLimiterConfig.getProps().entrySet()) {
+                result.append(indent(6)).append(each.getKey()).append(COLON_SPACE).append(each.getValue()).append(NEWLINE);
+            }
+        }
+    }
+    
+    private void getCommonScalingItemConfig(final String itemName, final ShardingSphereAlgorithmConfiguration itemConfig, final StringBuilder result) {
+        result.append(indent(3)).append(itemName).append(COLON_NEWLINE);
+        result.append(indent(4)).append("type").append(COLON_SPACE).append(itemConfig.getType()).append(NEWLINE);
+        if (null != itemConfig.getProps() && !itemConfig.getProps().isEmpty()) {
+            result.append(indent(4)).append("props").append(COLON_NEWLINE);
+            for (Map.Entry<Object, Object> each : itemConfig.getProps().entrySet()) {
+                result.append(indent(5)).append(each.getKey()).append(COLON_SPACE).append(each.getValue()).append(NEWLINE);
+            }
+        }
+    }
+    
+    private void getAlgorithmConfig(final Map<String, ShardingSphereAlgorithmConfiguration> algorithmMap, final StringBuilder result) {
+        for (Map.Entry<String, ShardingSphereAlgorithmConfiguration> each : algorithmMap.entrySet()) {
+            result.append(indent(2)).append(each.getKey()).append(COLON_NEWLINE);
+            result.append(indent(3)).append("type").append(COLON_SPACE).append(each.getValue().getType()).append(NEWLINE);
+            if (null != each.getValue().getProps() && !each.getValue().getProps().isEmpty()) {
+                result.append(indent(3)).append("props").append(COLON_NEWLINE);
+                for (Map.Entry<Object, Object> each1 : each.getValue().getProps().entrySet()) {
+                    result.append(indent(4)).append(each1.getKey()).append(COLON_SPACE).append(each1.getValue()).append(NEWLINE);
+                }
+            }
+        }
+    }
+    
+    private void getCommonDefaultStrategyConfig(final ShardingStrategyConfiguration ruleConfig, final StringBuilder result) {
+        if (ruleConfig instanceof StandardShardingStrategyConfiguration) {
+            StandardShardingStrategyConfiguration strategyConfig = (StandardShardingStrategyConfiguration) ruleConfig;
+            result.append(indent(2)).append("standard").append(COLON_NEWLINE);
+            result.append(indent(3)).append("shardingColumn").append(COLON_SPACE).append(strategyConfig.getShardingColumn()).append(NEWLINE);
+            result.append(indent(3)).append("shardingAlgorithmName").append(COLON_SPACE).append(strategyConfig.getShardingAlgorithmName()).append(NEWLINE);
```

Comment:
It would be better to declare these strings as global variables.  (standard, shardingColumn, shardingAlgorithmName, '-', ...)